force-app/main/default/classes/ApplicationTriggerHandler.cls

```diff
@@ -0,0 +1,315 @@
+public with sharing class ApplicationTriggerHandler {
+    
+    public static void afterInsert(List<Application__c> newApplications) {
```

Comment:
Make sure you commit all metadata that you created for this project. We won't be able to run tests and validate the objects/fields you created without them also being included.